src/pluto/image.cr

```diff
@@ -36,6 +36,33 @@ class Pluto::Image
     )
   end
 
+  private def each_channel(&)
+    yield @red, Channel::Red
+    yield @green, Channel::Green
+    yield @blue, Channel::Blue
+    yield @alpha, Channel::Alpha
+  end
+
+  private def [](ch : Channel)
+    case ch
+    when Channel::Red   then @red
+    when Channel::Green then @green
+    when Channel::Blue  then @blue
+    when Channel::Alpha then @alpha
+    else                     raise "Unknown channel #{ch} for Image"
```

Comment:
Do we need this `else`? It should already be impossible to pass anything, but a member of `Channel`

Reply:
Not yet, I was gonna introduce a new `Grey` channel with the `GreyImage` where this would be more relevant for the color image :sweat_smile: Want me to remove it?

Reply:
Let's move it to the PR introducing `Grey` and `GreyImage` then?